src/base/serial/sermouse.c

```diff
@@ -104,6 +104,11 @@ static void ser_mouse_move_buttons(int lbutton, int mbutton, int rbutton,
   add_buf(com, buf, sizeof(buf));
 }
 
+static void ser_mouse_move_wheel(int dy, void *udata)
+{
+  dosemu_error("serial wheel mouse\n");
```

Comment:
> I'd like to have PS/2 and serial modes supported as well

Please do, but this will require implementing
a 4-byte protocol instead of the current 3-byte.
Which AFAIK requires a lot of magic to activate
(mouse starts in compatibility mode, so you need
to support both protocols)